app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SignedOutController.scala

```diff
@@ -40,26 +41,39 @@ class SignedOutController @Inject()(
     extends AgentSubscriptionBaseController(authConnector, continueUrlActions, appConfig, subscriptionJourneyService)
     with SessionBehaviour {
 
-  def redirectToSos: Action[AnyContent] = Action.async { implicit request =>
+  def redirectTaskListUserToCreateCleanCreds: Action[AnyContent] = Action.async { implicit request =>
     withSubscribingAgent { agent =>
       for {
         agentSubContinueUrlOpt <- sessionStoreService.fetchContinueUrl
         continueId = agent.getMandatorySubscriptionRecord.continueId
       } yield {
-        val rootContinueUrl: String =
-          if (appConfig.isDevMode) "http://localhost:9437/agent-subscription/return-after-gg-creds-created"
-          else "/agent-subscription/return-after-gg-creds-created"
-        val continueUrl =
-          addParamsToUrl(
-            rootContinueUrl,
-            "id"       -> continueId,
-            "continue" -> agentSubContinueUrlOpt.map(_.url)
-          )
-        SeeOther(addParamsToUrl(appConfig.ggRegistrationFrontendExternalUrl, "continue" -> Some(continueUrl))).withNewSession
+        redirectToCreateCreds(agentSubContinueUrlOpt, continueId)
       }
     }
   }
 
+  def redirectUserToCreateCleanCreds: Action[AnyContent] = Action.async { implicit request =>
+    withAuthenticatedUser {
+      for {
+        agentSubContinueUrlOpt <- sessionStoreService.fetchContinueUrl
+      } yield {
+        redirectToCreateCreds(agentSubContinueUrlOpt, None)
+      }
+    }
+  }
+
+  private def redirectToCreateCreds(maybeContinueUrl: Option[ContinueUrl], continueId: Option[String]): Result = {
```

Comment:
to fit the pattern of the above methods should be named redirectTocreateCleanCreds

Reply:
Yep